include/nbl/builtin/glsl/blit/alpha_test/alpha_test.glsl

```diff
@@ -18,14 +18,14 @@ void nbl_glsl_blit_alpha_test_main()
 	const float alpha = nbl_glsl_blit_alpha_test_getPaddedData(ivec3(gl_GlobalInvocationID));
 
 	// Todo(achal): Need to pull this out in setData
-#if NBL_GLSL_EQUAL(_NBL_GLSL_BLIT_ALPHA_TEST_DIM_COUNT_, 1)
+#if NBL_GLSL_EQUAL(_NBL_GLSL_BLIT_DIM_COUNT_, 1)
 	#define LAYER_IDX gl_GlobalInvocationID.y
-#elif NBL_GLSL_EQUAL(_NBL_GLSL_BLIT_ALPHA_TEST_DIM_COUNT_, 2)
+#elif NBL_GLSL_EQUAL(_NBL_GLSL_BLIT_DIM_COUNT_, 2)
 	#define LAYER_IDX gl_GlobalInvocationID.z
-#elif NBL_GLSL_EQUAL(_NBL_GLSL_BLIT_ALPHA_TEST_DIM_COUNT_, 3)
+#elif NBL_GLSL_EQUAL(_NBL_GLSL_BLIT_DIM_COUNT_, 3)
 	#define LAYER_IDX 0
 #else
-	#error _NBL_GLSL_BLIT_ALPHA_TEST_DIM_COUNT_ not supported
+	#error _NBL_GLSL_BLIT_DIM_COUNT_ not supported
 #endif
```

Comment:
you could always use the Z-dimension of the dispatch workgroup count for texture layers

Reply:
Resolved.